lib/grandpa/types.go

```diff
@@ -62,6 +101,29 @@ func NewVoteFromHeader(h *types.Header) *Vote {
 	}
 }
 
+// NewVoteFromHash returns a new Vote given a hash and a blockState
```

Comment:
`...returns a new Vote given a hash and block header`

Reply:
it accepts a blockState?